koku/api/report/azure/openshift/query_handler.py

```diff
@@ -53,6 +59,72 @@ def __init__(self, parameters):
         super().__init__(parameters)
         # super() needs to be called before _get_group_by is called
 
+    @property
+    def annotations(self):
```

Comment:
I really like this idea. I think it might be better to put it in the ProviderMap class than in the QueryHandler, though.

My reasoning is to keep the QueryHandler focused on running the report query and letting the ProviderMap handle organizing the fields to be queried.

What do you think?